pkg/decoder/tagxl/v1/port152.go

```diff
@@ -6,17 +6,65 @@ import (
 	"github.com/truvami/decoder/pkg/decoder"
 )
 
+// Version: 1
+// +------+------+-----------------------------------------------+------------+
+// | Byte | Size | Description                                   | Format     |
+// +------+------+-----------------------------------------------+------------+
+// | 0    | 1    | version                                       | uint8      |
+// | 2    | 1    | new rotation state                            | uint8      |
+// | 2    | 1    | old rotation state                            | uint8      |
+// | 3    | 4    | timestamp in seconds since epoch              | time.Time  |
+// | 7    | 2    | number of rotations since last rotation       | float64    |
+// | 9    | 4    | elapsed seconds since last rotation           | uint32     |
+// +------+------+-----------------------------------------------+------------+
+//
+// Version: 2
+// +------+------+-----------------------------------------------+------------+
+// | Byte | Size | Description                                   | Format     |
+// +------+------+-----------------------------------------------+------------+
+// | 0    | 1    | version                                       | uint8      |
+// | 2    | 1    | sequence number                               | uint8      |
+// | 3    | 1    | new rotation state                            | uint8      |
+// | 3    | 1    | old rotation state                            | uint8      |
+// | 4    | 4    | timestamp in seconds since epoch              | time.Time  |
```

Comment:
`time.Time` is a go type. Please describe the raw payload, not what we parse it into, like in this case, it's a uint32 I believe